rnote-engine/src/strokes/textstroke.rs

```diff
@@ -689,6 +689,48 @@ impl TextStroke {
         }
     }
 
+    pub fn remove_word_before_cursor(&mut self, cursor: &mut GraphemeCursor) {
+        let cur_pos = cursor.cur_cursor();
+        let prev_pos = self.get_prev_word_start_index(cur_pos);
+
+        if cur_pos == prev_pos {
+            return;
+        }
+
+        cursor.set_cursor(prev_pos);
```

Comment:
this is redundant, no? if we use prev_pos below

Reply:
That's true. I'll also refactor the early return into an if block wrapping the rest of the body, like in the functions for removing a single grapheme.